app/cells/plugins/core/cell.rb

```diff
@@ -2,6 +2,10 @@ module Plugins
   module Core
     class Cell < FieldCell
       view_paths << "#{Cortex::Plugins::Core::Engine.root}/app/cells"
+
+      def required?
+        field_item.field.validations["presence"] == true
```

Comment:
We can just return `field_item.field.validations["presence"]` and nothing else and achieve the same effect - no need for a comparison